CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ endif()
 
 find_package(fastcdr REQUIRED )
 # new DDS API is required
-find_package(fastrtps 2 REQUIRED)
+find_package(fastdds 2 REQUIRED)
```

Comment:
```suggestion
find_package(fastdds 3 REQUIRED)
```

Reply:
I am going to apply the suggestion even though "2 required" should include the 3 (it should mean 2 or greater)

Reply:
That is true. The issue is that with your changes (changing the CMakes to require package fastdds) if you try to build with a version previous to Fast DDS v3, the compilation will fail. That's the reason behind major versions breaking API. And that is why, from this PR onwards, Fast DDS v3 is to be required.